script/deploy/_targetState.json

```
@@ -1556,9 +1554,38 @@
         "TokenWrapper": "1.0.0",
         "LiFiDEXAggregator": "1.0.0",
-        "Permit2Proxy": "1.0.0",
-        "GasZipPeriphery": "1.0.0",
-        "GasZipFacet": "2.0.0"
+        "Permit2Proxy": "1.0.2",
+        "GasZipPeriphery": "1.0.1",
+        "GasZipFacet": "2.0.2"
+      }
+    }
+  },
+  "sonic": {
```

Biome (1.9.4)

[error] The key sonic was already declared.

This where a duplicated key was declared again.

If a key is defined multiple times, only the last definition takes effect. Previous definitions are ignored.

(lint/suspicious/noDuplicateObjectKeys)